services/QuillLMS/client/app/bundles/Staff/components/locker/organizeLocker.tsx

```diff
@@ -115,7 +115,7 @@ export const OrganizeLocker = ({ history, personalLocker, userId }) => {
   }
 
   function handleAddSection() {
-    let updatedLockerPreferences = _.cloneDeep(lockerPreferences);
+    let updatedLockerPreferences = _.cloneDeep(lockerPreferences) || {};
```

Comment:
I should have caught this in initial review but I think this should actually be a const (you can still set properties to a constant hash).